pkr/cli/log.py

```diff
@@ -22,6 +23,10 @@ def write(msg, add_return=True, error=False):
     fd.flush()
 
 
+def write_fn(fn, msg):
+    os.write(fn, msg.encode("utf-8"))
```

Comment:
I doubt black will tolerate a 2 chars var name

Reply:
It is a pylint warning yes. But not in the mode we currently use.